charts/haproxy/templates/deployment.yaml

```diff
@@ -4,6 +4,10 @@ metadata:
   name: {{ include "haproxy.fullname" . }}
   labels:
     {{- include "haproxy.labels" . | nindent 4 }}
+  {{- if .Values.annotations }}
+  annotations:
+{{ toYaml .Values.annotations | indent 4 }}
```

Comment:
please align this line with the other ones, othewise it will become harder to read.